src/ru/company/onetwo33/Animal.java

```diff
@@ -0,0 +1,38 @@
+package ru.company.onetwo33;
+
+import java.util.Random;
+
+public abstract class Animal {
+
+    private static int COUNT_ANIMALS = 0;
+
+    private String name;
+    private int age;
+    private String color;
+
+    public Animal(String name, int age, String color) {
+        this.name = name;
+        this.age = age;
+        this.color = color;
+        COUNT_ANIMALS++;
+    }
+
+    public void run(double value) {}
+
+    public void swim(double value) {}
+
+    public void jump(double value) {}
```

Comment:
```suggestion
    public abstract void run(double value) {}

    public abstract void swim(double value) {}

    public abstract void jump(double value) {}
```
так как методы всегда переопределяются в дочерних классах